ui/apps/platform/cypress/fixtures/auth/TODO_ROX-12750.md

```diff
@@ -0,0 +1,4 @@
+The permissions `AllComments`, `ComplianceRuns`, `ComplianceRunSchedule`, `Config`, `DebugLogs`, 
+`NetworkGraphConfig`, `ProbeUpload`, `ScannerBundle`, `ScannerDefinitions`, `SensorUpgradeConfig` 
+and `ServiceIdentity` should be removed from the files `myPermissionsMinimalAccess.json` and 
+`myPermissionsNoAccess.json`.
```

Comment:
This file is here because JSON does not really support comments.
It should be removed when implementing ROX-12750.